README.md

```diff
@@ -239,6 +239,11 @@ Buildkite > (Org, Queue) > TotalAgentsCount
 
 When a queue is specified, only that queue's metrics are published.
 
+Jobs can be in one of the following states:
```

Comment:
```suggestion
We send metrics for Jobs in the following states:
```

We spoke a little bit publicly about the states a job can be in [here](https://www.buildkitestatus.com/incidents/k9nh8yy0px4q), this isn't a complete list of states, just the states we report metrics for.

Reply:
Actually here's the public documentation for [Job States](https://buildkite.com/docs/pipelines/defining-steps#job-states)